src/averageAges.js

```diff
@@ -15,11 +15,22 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(filterMen);
+  const result = men.reduce(averageAge, 0);
```

Comment:
Use a variable for initial value in reduce method (0) to improve readability. For example, const initialValue = 0; 📖